src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryUnary.cs

```diff
@@ -335,6 +341,30 @@ public static FormulaValue TextToBoolean(IRContext irContext, StringValue[] args
             return CommonErrors.InvalidBooleanFormatError(irContext);
         }
 
+        public static FormulaValue GetBoolean(IRContext irContext, FormulaValue value)
+        {
+            FormulaValue result = null;
+            switch (value)
+            {
+                case StringValue sv:
+                    result = TextToBoolean(irContext, new StringValue[] { sv });
+                    break;
+                case NumberValue num:
+                    result = NumberToBoolean(irContext, new NumberValue[] { num });
+                    break;
+                case BooleanValue boolVal:
+                    result = new BooleanValue(irContext, boolVal.Value);
```

Comment:
>new BooleanValue(irContext, boolVal.Value); [](http://example.com/codeflow?start=29&length=43)

nit - can this jus be:
result = boolVal;

Reply:
you right, i go around here. updated.